docs/commands.md

```diff
@@ -389,6 +390,23 @@ react-native run-ios [options]
 
 Builds your app and starts it on iOS simulator.
 
+### `build-ios`
+
+Usage:
+
+```sh
+react-native build-ios [options]
+```
+
+Builds IOS app.
+
+#### Options
+
+#### `--mode <release|debug>`
```

Comment:
Use `--configuration` instead of `--mode`, because `--mode` doesn't exist. 
```suggestion
#### `--configuration <release|debug>`
```

Reply:
Actually that was kinda requested feature to unify naming convention when building / running the app with custom params for both platforms.
You can read a bit more here: #1128  and here: #179